Library/Homebrew/utils/github.rb

```diff
@@ -522,8 +524,42 @@ def self.fetch_pull_requests(name, tap_remote_repo, state: nil, version: nil)
     []
   end
 
+  # WARNING: This returns results in a slightly different form from `fetch_pull_requests`.
+  def self.fetch_open_pull_requests(name, tap_remote_repo, version: nil)
+    return [] if tap_remote_repo.blank?
+
+    # Bust the cache every three minutes.
```

Comment:
Can we use an ETag or something to use the "proper" GitHub API cache handling here?

Reply:
It'll be a bit delicate, since doing that properly will affect a lot of other callers, but let me look into it.

Reply:
It can wait until another PR 👍🏻